subprojects/architecture-test/src/test/java/org/gradle/architecture/test/ArchUnitFixture.java

```diff
@@ -262,18 +280,17 @@ private void unpackJavaType(JavaType type, Set<JavaClass> referencedTypes, Set<J
         }
     }
 
-    class GradlePublicApi extends DescribedPredicate<JavaClass> {
+    class InGradlePublicApiPackages extends DescribedPredicate<JavaClass> {
         private static final PackageMatchers INCLUDES = PackageMatchers.of(parsePackageMatcher(System.getProperty("org.gradle.public.api.includes")));
         private static final PackageMatchers EXCLUDES = PackageMatchers.of(parsePackageMatcher(System.getProperty("org.gradle.public.api.excludes")));
-        private static final DescribedPredicate<JavaClass> TEST_FIXTURES = JavaClass.Predicates.belongToAnyOf(EmptyStatement.class, Matchers.class, PreconditionVerifier.class, Requires.class, SetSystemProperties.class, TestClassLoader.class, TestPrecondition.class, TestPreconditionExtension.class, UsesNativeServices.class, UsesNativeServicesExtension.class);
 
-        public GradlePublicApi() {
-            super("Gradle public API");
+        public InGradlePublicApiPackages() {
+            super("In Gradle public API packages");
```

Comment:
```suggestion
            super("in Gradle public API packages");
```
💄  When used directly, I suppose this should start with a lowercase letter.

Reply:
Done